docs/blog/2025-08-mcp-langchain-with-stepflow.md

```diff
@@ -0,0 +1,369 @@
+---
+date: 2025-08-27
+title: "Building an AI-Powered Research Assistant with Stepflow: Orchestrating LangChain and MCP"
+description: "How to combine AI processing with practical tool integration using declarative workflows"
+slug: mcp-langchain-with-stepflow
```

Comment:
Maybe mcp-research-assistant? (Nice to have *what* we're showing in the slug)

Reply:
Good call.